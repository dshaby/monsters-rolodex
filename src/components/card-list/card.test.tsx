import { render } from "@testing-library/react";
import Card from "./card.component";

const myMonster = {
    id: "123",
    name: "Daniel",
    email: "hey@me.com"
}

test("Card Component renders", () => {
    // my tests
    const {asFragment} = render(<Card monster={myMonster} />);
    // expect((screen.getByRole("div").length).)
    expect(asFragment()).toMatchSnapshot();
});
