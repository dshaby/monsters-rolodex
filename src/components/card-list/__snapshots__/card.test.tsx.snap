// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders 1`] = `
<DocumentFragment>
  <div
    class="card-container"
  >
    <img
      alt="monster Daniel"
      src="https://robohash.org/123?set=set2&size=180x180"
    />
    <h2>
      Daniel
    </h2>
    <p>
      hey@me.com
    </p>
  </div>
</DocumentFragment>
`;
